OpenOversight/tests/conftest.py

```diff
@@ -310,15 +339,18 @@ def add_mockdata(session):
 
     test_images = [
         models.Image(
-            filepath="/static/images/test_cop{}.png".format(x + 1), department_id=1
+            filepath="/static/images/test_cop{}.png".format(x + 1),
```

Comment:
🔬 An f-string feels like a more intuitive way to write this, imo.

Reply:
Strongly agree, this is from old code. Changed this one. And maybe we can use https://github.com/ikamensh/flynt or something next to get rid of all of the `.format`.